locustfiles/upload_dsslocust.py

```diff
@@ -54,3 +54,9 @@ class UploadUser(DSSLocust):
     max_wait = 500
     task_set = [UploadLocalTaskSet, UploadCloudTaskSet]
     weight = 1
+
+    def use_scale_settings(self):
```

Comment:
Where is this used?

Reply:
The same place the `scale_test` directory is used.  Nowhere.  XD  Was just consolidating code, but really the whole thing should be considered for deletion until it's used I believe.